homeassistant/components/hifiberry/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+    "title": "hifiberry",
+    "config": {
+        "step": {
+            "user": {
+                "data": {
+                    "host": "[%key:common::config_flow::data::host%]",
+                    "port": "[%key:common::config_flow::data::port%]",
+                    "authtoken": "authtoken"
```

Comment:
The value on the right: Should it be "Auth Token"? to match style used in other config flows.

Alternatively, could consider a name like `access_token` -- even if it is not quite the same terminology used in the API.